webapp/channels/src/i18n/en.json

```diff
@@ -2724,44 +2725,44 @@
   "admin.support.termsOfServiceTitle": "Custom Terms of Service",
   "admin.support.termsTitle": "Terms of Use Link:",
   "admin.system_properties.confirm.delete.button": "Delete",
-  "admin.system_properties.confirm.delete.text": "Deleting this property will remove all user-defined values associated with it.",
-  "admin.system_properties.confirm.delete.title": "Delete {name} property",
+  "admin.system_properties.confirm.delete.text": "Deleting this attribute will remove all user-defined values associated with it.",
+  "admin.system_properties.confirm.delete.title": "Delete {name} attribute",
   "admin.system_properties.details.saving_changes": "Saving configuration…",
   "admin.system_properties.details.saving_changes_error": "There was an error while saving the configuration",
-  "admin.system_properties.user_properties.add_property": "Add property",
+  "admin.system_properties.user_properties.add_property": "Add attribute",
   "admin.system_properties.user_properties.dotmenu.ad_ldap.edit_link.label": "Edit link with: <Chip>AD/LDAP: {propertyName}</Chip>",
-  "admin.system_properties.user_properties.dotmenu.ad_ldap.link_property.label": "Link property to AD/LDAP",
-  "admin.system_properties.user_properties.dotmenu.delete.label": "Delete property",
-  "admin.system_properties.user_properties.dotmenu.duplicate.label": "Duplicate property",
+  "admin.system_properties.user_properties.dotmenu.ad_ldap.link_property.label": "Link attribute to AD/LDAP",
+  "admin.system_properties.user_properties.dotmenu.delete.label": "Delete attribute",
+  "admin.system_properties.user_properties.dotmenu.duplicate.label": "Duplicate attribute",
   "admin.system_properties.user_properties.dotmenu.duplicate.name_copy": "{fieldName} (copy)",
   "admin.system_properties.user_properties.dotmenu.saml.edit_link.label": "Edit link with: <Chip>SAML: {propertyName}</Chip>",
   "admin.system_properties.user_properties.dotmenu.saml.link_property.label": "Link property to SAML",
```

Comment:
This one got missed earlier:
  "admin.system_properties.user_properties.dotmenu.saml.link_property.label": "Link attribute to SAML",